### ISSUE TITLE
Cannot set network.http.useToken to true via xml config

### ISSUE THREAD

**Issue by jpstrikesback** [orientechnologies/orientdb#6633]
Also see: #6478 

## OrientDB Version, operating system, or hardware.
- v2.2.7 (Docker)

ORIENTJS
- v2.2.2

## Operating System
- [ ] Linux

## Expected behaviour and actual behaviour
The following config in orientdb-server-config.xml appears to accomplish nothing
``` xml
    <properties>
        <entry name="network.http.useToken" value="true"/> 
    </properties>
```

relevant config settings from console `orientdb>config` command result:

```
|109 |network.http.sessionExpireTimeout                  |300                                                                                            |
|110 |network.http.useToken                              |false                                                                                          |
|111 |network.token.secretyKey                           |                                                                                               |
|112 |network.token.encriptionAlgorithm                  |HmacSHA256                                                                                     |
|113 |network.token.expireTimeout                        |60                                                                                             |
```

I can use config set on the console to enable but otherwise adding the above in the server config doesn't appear to set these props.

I am trying to enable tokens as I am getting frequent errors like this in OrientJS:

```
{ OrientDB.RequestError: The token provided is expired
    at Operation.parseError (/usr/local/lib/node_modules/.../node_modules/orientjs/lib/transport/binary/protocol33/operation.js:864:13)
    at Operation.consume (/usr/local/lib/node_modules/.../node_modules/orientjs/lib/transport/binary/protocol33/operation.js:455:35)
    at Connection.process (/usr/local/lib/node_modules/.../node_modules/orientjs/lib/transport/binary/connection.js:399:17)
    at Connection.handleSocketData (/usr/local/lib/node_modules/.../node_modules/orientjs/lib/transport/binary/connection.js:290:20)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:177:18)
    at Socket.Readable.push (_stream_readable.js:135:10)
    at TCP.onread (net.js:542:20)
  name: 'OrientDB.RequestError',
  message: 'The token provided is expired',
  data: {},
  previous: [],
  id: 1,
  type: 'com.orientechnologies.orient.enterprise.channel.binary.OTokenSecurityException',
  hasMore: 0 }
```

**Comment by maggiolo00** [comment]
@jpstrikesback 

how do you configure the Connection?

do you use

```
useToken: true
```

**Comment by jpstrikesback** [comment]
Indeed I do :)

**Comment by maggiolo00** [comment]
try to increase the token timeout

http://orientdb.com/docs/2.2/Configuration.html#networktokenexpiretimeout

**Comment by jpstrikesback** [comment]
I had it set to 360000 I believe, but that was another setting that didn't
register

**Comment by maggiolo00** [comment]
Can you check the config  in Studio?

```
http://<ip>:<httpPort>/studio/index.html#/dashboard/general
```

then Global Cfg tab

Thanks

**Comment by jpstrikesback** [comment]
Ok, so, the timeout (and key) were settings I added after initially reporting:

``` xml
    <properties>
        <entry value="1" name="db.pool.min"/>
        <entry value="50" name="db.pool.max"/>
        <entry value="false" name="profiler.enabled"/>
        <entry name="network.http.useToken" value="true"/>
        <entry name="network.token.secretKey" value="...."/>
        <entry name="network.token.expireTimeout" value="60000"/>
    </properties>
```

They still didn't & don't show up in the console config list BUT I haven't had the error since I added them and the dash shows that the settings are enabled (the console still shows them as disabled)

On dash:

<img width="1470" alt="screenshot 2016-09-01 12 31 00" src="https://cloud.githubusercontent.com/assets/445563/18175691/2fcf2d36-7040-11e6-9487-c54d0a7ea484.png">

I'd like to narrow it down but since it's working I've run out of time to play. Feel free to leave this open or close it until I can repeat it.

Cheers!
JP
